indico/htdocs/sass/partials/_inputs.scss

```diff
@@ -830,3 +830,44 @@ dd > .i-has-action {
         padding: 0.25em 0;
     }
 }
+
+
+.color-palette {
+    background: #FFF;
+    background-clip: padding-box;
+    border: thin solid rgba(0, 0, 0, .15);
```

Comment:
Is the alpha channel actually needed or is this just a way of getting a shade of gray? (͡° ͜ʖ ͡°)